app/controlplane/pkg/data/ent/schema/workflow.go

```diff
@@ -46,11 +46,17 @@ func (Workflow) Fields() []ent.Field {
 			Annotations(&entsql.Annotation{
 				Default: "CURRENT_TIMESTAMP",
 			}),
+		field.Time("updated_at").
+			Default(time.Now).
+			Annotations(&entsql.Annotation{
+				Default: "CURRENT_TIMESTAMP",
+			}),
 		field.Time("deleted_at").Optional(),
 		// public means that the workflow runs, attestations and materials are reachable
 		field.Bool("public").Default(false),
 		field.UUID("organization_id", uuid.UUID{}),
 		field.UUID("project_id", uuid.UUID{}),
+		field.UUID("latest_run", uuid.UUID{}).Optional().Nillable(),
```

Comment:
what does nillable mean compared to optional?

Reply:
- `Optional`: Indicates that the field is not required when creating or updating an entity. The field can be omitted in the input data.
- `Nillable`: Indicates that the field can hold a `nil` (or `null` in SQL) value in the database.